### ISSUE TITLE
error after running make

### ISSUE THREAD

**Issue by bmeznarsic** [evolve-os/budgie-desktop#143]
OS: elementary OS 0.3 Freya beta1
I ran ./configure --disable-wm and make... 
Output here: http://hastebin.com/uxewosiyar.hs

**Comment by ikeydoherty** [comment]
Which Vala version  ?

**Comment by Justinzobel** [comment]
elementary's libmutter version will always cause havoc with budgie just FYI.

**Comment by bmeznarsic** [comment]
@ikeydoherty Installed: 0.26.2-0ubuntu1~14.04~valateam0
@Justinzobel I disabled wm at config...

**Comment by ikeydoherty** [comment]
Yeah this isn't WM related sadly, GTK version in Elementary ?

(Sidenote: Fixed WM cruft)

**Comment by bmeznarsic** [comment]
@ikeydoherty Installed: 3.14.7-0ubuntu1~14.04~ricotz0

**Comment by ikeydoherty** [comment]
Very odd, we specifically check the gtk version is > 313, and work around it:

```
# Check for GTK3.13+ (Le World Blow Up)
PKG_CHECK_MODULES(GTK313, [gtk+-3.0  >= 3.13], [have_gtk313=yes], [have_gtk313=no])
if test "x$have_gtk313" = "xyes" ; then
    VALAFLAGS="$VALAFLAGS --define HAVE_GTK313"
fi
```

```
#if HAVE_GTK313
    protected override Gtk.WidgetPath get_path_for_child(Gtk.Widget child)
    {
        Gtk.WidgetPath path = base.get_path_for_child(child);
#else
```

**Comment by bmeznarsic** [comment]
Welp, we did get the gtk+ version bump recently. Two weeks ago I was still on gtk+ 3.12, when testing... Dunno what made the difference but I just nuked the install and set everything up again. Again. Disabled the wm and can now run the panel in Pantheon's session.
![screenshot from 2015-01-24 16 00 49](https://cloud.githubusercontent.com/assets/6306479/5887739/3aeb621a-a3e2-11e4-9e43-d36ecf68fd3c.png)
